charmcraft/commands/store/__init__.py

```diff
@@ -57,16 +56,60 @@ class WhoamiCommand(BaseCommand):
     help_msg = "returns your login information relevant to the store"
 
     _titles = [
-        ('name:', 'display-name'),
+        ('name:', 'name'),
         ('username:', 'username'),
-        ('id:', 'id'),
+        ('id:', 'userid'),
     ]
 
     def run(self, parsed_args):
         """Run the command."""
-        client = Client()
-        result = client.get('/v1/whoami')
+        store = Store()
+        result = store.whoami()
 
         longest_title = max(len(t[0]) for t in self._titles)
-        for title, key in self._titles:
-            logger.info("%-*s %s", longest_title, title, result[key])
+        for title, attr in self._titles:
+            logger.info("%-*s %s", longest_title, title, getattr(result, attr))
```

Comment:
as you're touching this code anyway, and have pulled in `tabulate`, please switch this to use tabulate.
it's a lot easier to follow; the pesky `_titles` goes away and the meat of the above becomes just a call to `tabulate`,
```python
logger.info(tabulate([
    ['name:', result.name],
    ['username:', result.username],
    ['id:', result.userid],
], tablefmt='plain')
```
and that's it!  :grin:

Reply:
Great idea! I had to get the table and log each line separatedly, of course, but the "table building" is done now by tabular. Thanks!